docs/components/List/List.stories.mdx

```diff
@@ -172,7 +178,7 @@ and no border-bottom. You can also experiment by toggling this prop in the other
 custom render examples to observe its impact.
 
 Note: The `customItemStyles` prop is only valid when a `customRenderItem`
-function is provided.
```

Comment:
This makes it sound to me like `customSectionStyles` is only valid if `customRenderItem` is provided but I believe it is only valid if `customRenderSection` is provided?